crates/rpc/rpc-api/src/trace.rs

```diff
@@ -17,6 +17,7 @@ pub trait TraceApi {
         call: CallRequest,
         trace_types: HashSet<TraceType>,
         block_id: Option<BlockId>,
+        block_overrides: Option<Box<BlockOverrides>>,
```

Comment:
hmm, I think we should also add state overrides, mimicking `eth_call`.
because if we only add block overrides then adding state overrides later will be an issue if we want them before the block overrides

Reply:
@mattsse Done